debian/changelog

```diff
@@ -1,3 +1,9 @@
+matrix-synapse-py3 (1.65.0+nmu1) stable; urgency=medium
+
+  * Allocate a group for the system user.
+
+ -- Synapse Packaging team <packages@matrix.org>  Tue, 23 Aug 2022 17:17:00 +0100
```

Comment:
this should be your name and email, not ours.

Reply:
Fixed. This might warrant a note in the contributing guide, though. I went with that, because there hasn't been anyting else for a few years now.

Reply:
well, it's what `dch` gives you :)